src/routes/contents/[id]/+page.svelte

```diff
@@ -0,0 +1,229 @@
+        }
+        return title;
+    }
+
+    function handleBackNavigation(event) {
+        event.preventDefault();
+        if ($contentsStack.length > 0) {
+            const menuId = contentsStack.popItem();
+            goto(`${base}/contents/${menuId}`);
+        }
+    }
+    $: showBackButton = $contentsStack.length > 0;
+</script>
+
+<div class="grid grid-rows-[auto,1fr]" style="height:100vh;height:100dvh;">
+    <div class="navbar">
+        <Navbar on:backNavigation={handleBackNavigation} {showBackButton}>
+            <!-- <div slot="left-buttons" /> -->
+            <label for="sidebar" slot="center">
+                <div class="btn btn-ghost normal-case text-xl">{title}</div>
+            </label>
+            <!-- <div slot="right-buttons" /> -->
+        </Navbar>
+    </div>
+
+    <div class="overflow-y-auto mx-auto max-w-screen-md">
+        <div id="container" class="contents" style={convertStyle($s['body.contents'])}>
+            {#each $page.data.items as item}
+                <!-- iterate through the items, add html -->
+
+                <!-- svelte-ignore a11y-invalid-attribute -->
+                <a
+                    href="#"
+                    class="contents-link contents-link-ref"
+                    on:click={(event) => onClick(event, item)}
+                >
+                    <div class="contents-item-block" id={item.id}>
+                        <!--check for the various elements in the item-->
+
+                        {#if item.imageFilename}
+                            <div
+                                class="contents-image-block"
+                                style="{convertStyle(
+                                    $s['div.contents-image-block']
+                                )}{checkImageSize(item)}"
+                            >
+                                <img
+                                    class="contents-image"
+                                    src="{base}/illustrations/{item.imageFilename}"
+                                    alt={item.imageFilename}
+                                />
+                            </div>
+                        {:else}
+                            <!--empty - no image-->
+                        {/if}
+
+                        <div class="contents-text-block">
+                            <!-- check for title -->
+                            {#if $page.data.features['show-titles'] === true}
+                                {#if item.title.default === undefined}
+                                    <!--empty-->
+                                {:else}
+                                    <div class="contents-title">{item.title.default}</div>
+                                {/if}
+                            {/if}
+
+                            <!--Check for subtitle-->
+                            {#if $page.data.features['show-subtitles'] === true}
+                                {#if item.subtitle.default === undefined}
```

Comment:
You can change this so that it a positive check:

`{#if item.subtitle.default }`

and put the div in the if condition and remove the else.